documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -7,6 +7,13 @@
 * [solution_name] now supports binary scanning of files up to 100GB via a chunking method employed during upload.    
     <note type="note">This feature requires [blackduck_product_name] 2024.7.0 or later.</note>
 
+### Dependency updates
+
+* Component Location Analysis version updated to 1.1.13.
+* Logback Core version updated to 1.2.13.
+
 ### Changed features
 
-### Resolved issues
\ No newline at end of file
+### Resolved issues
+
+* (IDETECT-4408) - Remediated vulnerability in Logback-Core library.
```

Comment:
Please add the CVE links.
https://nvd.nist.gov/vuln/detail/CVE-2023-6378
https://nvd.nist.gov/vuln/detail/CVE-2023-6481

Reply:
Done, thanks!